lib/challenge.js

```diff
@@ -179,11 +208,19 @@ function prompt9() {
  */
 function prompt10() {
   const myArray = [];
-  // YOUR CODE HERE
+  for (let i =1; i<100; i++){
+    let isEven = i % 2 == 0
+    if (isEven){
+      myArray.push(i)
+      return myArray;
+      // ***  getting illegal return statement error in browser
+      console.log(myArray[i]);
```

Comment:
A `return` exits the function! So you cannot have logic below such as line 217, it will not run! ☺️